.github/workflows/sonar-step-analyze.yml

```diff
@@ -46,17 +46,7 @@ jobs:
         uses: actions/download-artifact@v3
         with:
           name: ${{inputs.ARTIFACT_FILENAME}}
-          path: ${{ inputs.WORKING_DIRECTORY }}/${{ inputs.ARTIFACT_PATH }}${{inputs.ARTIFACT_FILENAME}}
-
-      - name: Check lcov in wrong position file existence (workaround)
-        id: check_lcov_file
-        uses: andstor/file-existence-action@v1
-        with:
-          files: "coverage/lcov.info/lcov.info"
-
-      - name: lcov in wrong position file exists (workaround)
-        if: steps.check_lcov_file.outputs.files_exists == 'true'
-        run: mv coverage/lcov.info/lcov.info ./lcov.info && rm -rf coverage/lcov.info && mv ./lcov.info ./coverage/lcov.info
+          path: ${{ inputs.WORKING_DIRECTORY }}/${{ inputs.ARTIFACT_PATH }}
```

Comment:
Perchè togliere tutto il check del file che esiste dovrebbe fixare? Cosa succede se su un progetto non ho la coverage perchè non ho i test?

Reply:
Se su un progetto non c'è la coverage in questo momento questo job fallisce comunque perché il download-artifact fallisce se non trova un artefatto.

Reply:
In ogni caso il fix è rimuovere `${{inputs.ARTIFACT_FILENAME}} che era ridondante perché viene già scaricato con il nome usato come input.

La rimozione degli altri step è stata fatta perché erano un workaround ed in ogni caso non funzionavano.